www-client/qutebrowser/qutebrowser-2.5.3.ebuild

```
@@ -128,6 +128,11 @@ python_install_all() {
 	fperms -x /usr/share/${PN}/{scripts/cycle-inputs.js,userscripts/README.md}
 	python_fix_shebang "${ED}"/usr/share/${PN}
 
+	if use apparmor; then
+		insinto /etc/apparmor.d
+		doins misc/apparmor/*
```

```suggestion
		doins misc/apparmor/usr.bin.${PN}
```
I prefer to avoid globs and be more explicit where possible. In this case it feels very unlikely to ever get new files too, albeit I'd notice them given I check added/removed source files on bump.

---

It is unlikely, but it will be annoying when we do miss an update, since on most systems this will result in silent lack of AppArmor's protection of some executable.

---

Well, I don't mind keeping the directory if you feel concerned. Albeit I prefer `doins -r misc/apparmor/.` rather than use globs still.